breathecode/assignments/tests/urls/tests_user_me_task.py

```diff
@@ -163,6 +163,32 @@ def test_user_me_task__with_two_task(self):
         self.assertEqual(response.status_code, status.HTTP_200_OK)
         self.assertEqual(self.bc.database.list_of('assignments.Task'), self.bc.format.to_dict(model.task))
 
+    """
+    🔽🔽🔽 Get with querystring assets
+    """
+
+    @patch('breathecode.assignments.signals.assignment_status_updated.send', MagicMock())
+    @patch('django.db.models.signals.pre_delete.send', MagicMock(return_value=None))
+    @patch('breathecode.admissions.signals.student_edu_status_updated.send', MagicMock(return_value=None))
```

Comment:
this still doesn't prove that this works, because it does not have the case where it does not get results